### ISSUE TITLE
hexo generates empty files

### ISSUE THREAD

**Issue by honkiko** [hexojs/hexo#4268]
## Check List

Please check followings before submitting a new issue.

- [y ] I have already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting)
- [y ] I have already searched existing issues and they are not help  to me
- [ y] I examined error or warning messages and it's difficult to solve
- [y ] Using [the latest](https://github.com/hexojs/hexo/releases) version of Hexo (run `hexo version` to check)
- [ y] Node.js is higher than 8.6.0

## Expected behavior
generates site like before
## Actual behavior
generates emtpy files
## How to reproduce?

I have gh pages repo https://github.com/TencentCloudContainerTeam/TencentCloudContainerTeam.github.io, and build from develop branch by travis-ci.org.  "hexo g" is called from .travis.yml(https://github.com/TencentCloudContainerTeam/TencentCloudContainerTeam.github.io/blob/develop/.travis.yml).

The CI setup works for last 2 years. But yesterday empty files are pushed into master branch.
The travis-ci build log: https://api.travis-ci.org/v3/job/679426727/log.txt

I'm not sure the travis online log is accessible by others. So I also uploaded a copy.
[travis-hexo-generates-empty-files.log](https://github.com/zalando-incubator/hexo-theme-doc/files/4533343/travis-hexo-generates-empty-files.log)

## Environment & Settings

**Node.js & npm version**

```
node v14.0.0
npm v6.14.4
```

**Your site `_config.yml`** (Optional)

```
https://github.com/TencentCloudContainerTeam/TencentCloudContainerTeam.github.io/blob/develop/_config.yml
```

**Your theme `_config.yml`** (Optional)

```
https://github.com/TencentCloudContainerTeam/TencentCloudContainerTeam.github.io/blob/develop/themes/landscape/_config.yml
```

**Hexo and Plugin version(`npm ls --depth 0`)**

```
travis-ci.org
```

**Your package.json `package.json`**

```
https://github.com/TencentCloudContainerTeam/TencentCloudContainerTeam.github.io/blob/develop/package.json
```

**Comment by stevenjoezhang** [comment]
Please downgrade to Node.js 13 or 12.

https://github.com/hexojs/hexo/issues/4267#issuecomment-619394907

**Comment by dumindu** [comment]
@stevenjoezhang Using v 12.5.0 via nvm on mac. But still having same issue while running `hexo/site`. Any other settings/ package version I need to change?
Thanks

**Comment by asad-awadia** [comment]
@dumindu I had to downgrade to 10.19.0

**Comment by dumindu** [comment]
I think this issue is fixed even in current version. Check the implementation of hexo site repo.

**Comment by asad-awadia** [comment]
@dumindu version of what? Node or hexo?

**Comment by shawumu** [comment]
is this problem solved? I was encountered the same issue.I've tried hexo version 3.2~3.9,and it does not work. hexo generate every file with empty content.

**Comment by stevenjoezhang** [comment]
Hexo 3.9 is outdated. Install version 5.4.0

**Comment by edgexie** [comment]
> Hexo 3.9 is outdated. Install version 5.4.0

thanks , it's help me

**Comment by khiav223577** [comment]
Update hexo from `v3.8.0` to `v6.3.0` solved my problem (Note that `v3.9.0` doesn't work)

### Update command:

```bash
npm install hexo@6
```

### My Environment
- npm: v8.19.3
- node: v16.20.1
